pyproject.toml

```diff
@@ -261,16 +261,6 @@ reportPrivateUsage = false
 reportUnknownMemberType = false
 reportUnnecessaryCast = false  # Unnecessary "cast" call; type is already...
 
-[tool.autopep8]
-max_line_length = 99
-in-place = true
-recursive = true
-hang-closing = true
-aggressive = 2
```

Comment:
Not sure what these lines were doing and if they would still be needed with `ruff-format`